src/sdgio-edit.owl

```diff
@@ -158,6 +159,62 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
 Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020236>))
 Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020237>))
 Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020300>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020308>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020309>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020310>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020311>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020312>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020313>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020315>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020317>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020318>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020319>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020320>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020325>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020326>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020327>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020328>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020329>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020331>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020334>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020335>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020337>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020338>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020339>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020341>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020342>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020343>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020349>))
```

Comment:
The definition and label are basically the same here, with no supporting semantics to help. I've tried to expand the def slightly, but we need more input on what this actually entails.

Reply:
You are also no using the genus/differentia phrasing in these defs, leading to a lot of repetition. Please fix that globally.

Reply:
New definition: "A planned process during which an agent or group of agents improves the policies of some organization O so that O increases its ability to contribute to some collective planned process that a group of organizations to which O belongs implements." The SRF document and other CGIAR documents that I know are not more specific. Do you want me to open an issue?

Reply:
> You are also not using the genus/differentia phrasing in these defs, leading to a lot of repetition. Please fix that globally.

This is not addressed.

> "A planned process during which an agent or group of agents improves the policies of some organization O so that O increases its ability to contribute to some collective planned process that a group of organizations to which O belongs implements."

We're not going to use variables like "O" in SDGIO defs as the audience is not familiar with these.

planned process to improve organizational policies to increase the capacity of a partnership of organisations to work towards a common objective
"A planned process to improve organizational policies during which an agent increases its ability to contribute to a planned process collectively pursued by a group of organisations "

Reply:
Thanks for rephrasing the definition. 

About the genus/differentia: I thought the way to apply that style in the definition of processes was to say _An A during which B happens_. Is that not the case? I thought that, among other places, that was said in issue #183.

Reply:
Rephrased definitions with the genus being the immediate superclass.